### PR TITLE
Updated release notes

### PR DESCRIPTION
**Pull request by Anne56njeri**
Updated the AzureAppConfiguration Task and AzureAppConfigurationPushTask release notes.

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: 
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+
+### v3.3.16 - February, 16 2021
+* Fixed “task.json” as the task editor gives wrong assistance.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md:8]: Can you reference the issue that prompted this (#447). Also, task.json is an implementation detail no need to mention it. The meaningful information for users is more like "Fixed incorrect tooltip help messages for task parameters in task editor UI."

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v1.2.4 - February, 16 2021
+* Fixed “task.json” as the task editor gives wrong assistance. 
+* Added error message when certificate based-authentication is used, as its not supported.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:7]: The hyphen in "based-authentication" should be removed.

Also, rewrite suggestion if you want "Updated error message when certificate based authentication is used to indicate that it is not supported."

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v1.2.4 - February, 16 2021
+* Fixed “task.json” as the task editor gives wrong assistance. 
+* Added error message when certificate based-authentication is used, as its not supported.
+* Fixed yaml file parsing.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:8]: Can we be a bit more helpful here. Ideally if we look back in a year we could tell a bit about what the problem was. It was an issue with flattening yaml objects right?

Comment (Anne56njeri) [reply releaseNotes/AzureDevOpsPushPipelineExtension.md:8]: Yes, it was an issue with flattening objects. We don't have a GitHub issue for this one

**Review by jimmyca15 — APPROVED** [review]

**Pull request merged**